### DISCUSSION TITLE
Unfortunately, I cannot change the mode instantly depending on the mode of the system while the application is open.

### DISCUSSION THREAD

**Discussion by kubilaybzk** [nativewind/nativewind#315] [General]
Unfortunately, I cannot change the mode instantly depending on the mode of the system while the application is open. When the application is opened, it works correctly if the phone is in dark or light color mode, but if the phone changes mode to dark mode or vice versa while the application is open, unfortunately the theme does not change, is there a way to do this?

**Comment by marklawlor** [comment]
NativeWind uses `appearance.addChangeListener` https://reactnative.dev/docs/appearance#addchangelistener to listen to the color scheme change. Can you please verify that it is working correctly?

If it does not, please sure you have followed followed the setup guide for your framework (eg Expo Apps need a config change & an additional package https://docs.expo.dev/guides/color-schemes/)

If it does, please either follow the troubleshooting guide https://www.nativewind.dev/guides/troubleshooting or try the `next` branch #308

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
